src/emulator/glutil/CMakeLists.txt

```diff
@@ -9,3 +9,4 @@ add_library(
 
 target_include_directories(glutil PUBLIC include)
 target_link_libraries(glutil PUBLIC glbinding)
+target_link_libraries(glutil PUBLIC util spdlog)
```

Comment:
Futureproofing? I just added it everywhere so people don't have to think about it if/when they need to log stuff from anywhere. Same goes for `disasm`.

I'll delete it and it can be added if you think so too.